docs/eventTrigger.rst

```diff
@@ -0,0 +1,156 @@
+.. _EventBasedTrigger:
+
+
+Flow Trigger Dependency Plugin
+==================================
+*****
+Event Based Trigger
+*****
+
+
+..
+   Todo:: Link to the data trigger documentation if available
+Currently there are only few ways to launch jobs in Azkaban including schedules and API. However, they are limited because sometimes jobs need to be executed automatically on demand. Event trigger is a new feature introduced by Azkaban. It defines a new paradigm of triggering flows - triggering a flow on event arrival. This concept enables users to define events that the flow depends on. Once all of the dependencies become ready, a workflow will be triggered. 
```

Comment:
Azkaban supports launching flows via scheduling it or Ajax API.

Reply:
triggering a flow on event arrival -> triggering a flow on kafka event arrival

Reply:
This concept enables users to define events that the flow depends on. Once all of the events become ready, a workflow will be triggered.